### DISCUSSION TITLE
SolisCloud + CT/PV info

### DISCUSSION THREAD

**Discussion by barthoefs** [hultenvp/solis-sensor#238] [Q&A] [answered]
Hello,
Is there a possibility to read out the CT clamp info on a Solis RAI-3K-48ES-5G? This is only a battery inverter, not hybrid, not PV. It has a meter to read the grid (this is working fine). It also has a extra clamp, connected on the CT port of the inverter, to read out the external PV devices. This is the way Solis can make a correct graph between battery/inverter/grid => usage.
Thanks

**Comment by hultenvp** [comment]
Hi @barthoefs,

Are you using m.ginlong.com or soliscloud?
In principle, if the API had the attribute you need then I can add it.

For m.ginlong.com we need to find out what your API renturns, there's no spec: 
* enable debug and remove the coment line in line 306 of ginlong_api.py (#_LOGGER.debug("Payload = %s", payload))
* Reboot and it will output the repsonse from ginlong.com. 
* You can use online tools like https://jsonformatter.org/json-pretty-print, to get the json readable. 
* Then you can search for the value you'd like me to add

For soliscloud.com you'll have to look in the API spec (https://github.com/hultenvp/solis-sensor/blob/master/doc/SolisCloud%20API%201.2.pdf)
If you can find the attribute, it can be added. 

Cheers

**Comment by barthoefs** [comment, marked as answer]
@hultenvp seems it was related to it not showing SolisCloud app/site.
Since last week Solis AC Output Total Power is reporting the CT clamp again.
